jq-defect-2020-1/README.md

```diff
@@ -0,0 +1,72 @@
+# jq Use-After-Free
+
+[JQ](https://github.com/stedolan/jq) is a command line utility and popular C
+library for parsing and manipulating JSON.
+
+## Build
+
+You can build or pull the tag `forallsecure/jq-defect-2020-1` as below:
+
+```bash
+docker pull -t forallsecure/jq-defect-2020-1 .
+```
+
+## Running
+
+Run using the Dockerfile-defined entrypoint:
+
+```bash
+docker run --rm -it forallsecure/jq-defect-2020-1 .
+```
+
+The time to find the crash is typically under a minute, though it can vary by
+several minutes due to the large potential state space to explore. It should
+be found within 10 minutes when run with the container entrypoint on a single
+core.
+
+## Defect Found: Use-After-Free
+
+The `poc/` directory contains a seed which triggers a heap use-after-free as
+detected by ASAN. This bug was reported to the maintainers of jq on 14 Feb
```

Comment:
Lets mention Harrison here too ("...reported by Harrison, a ForAllSecure intern, to the..." etc)